### PR TITLE
Added rankings

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

File: lib/tournament.json [hunk 1]
```diff
@@ -625,6 +625,44 @@
       }
     ]
   ],
+  "rankings": [
```
Comment (john-mikhail) [inline lib/tournament.json]: Wonder if for the sake of simplicity in the FE, would combining all the information regarding a team together with the team declaration be easier.  This way I don't need to do mappings of teams to extra data.

So essentially two structures...teams/round-robin.  The teams part contains the name, members, rankings, etc.  and round-robin shows the upcoming matches and whether they're completed or not?

Comment (pylaligand) [inline lib/tournament.json]: The reason that I'm doing it this way is that the "teams" field always remains unchanged but "round-robin" and "rankings" get constantly updated. Ultimately the latter two will end up in a database whereas the former will be encoded in a file.

Comment (pylaligand) [inline lib/tournament.json]: Btw this is not generated manually, I have a script that generates the data but I'll check it in later to focus on delivering the data for now.

Comment (john-mikhail) [inline lib/tournament.json]: Fair enough.  I could do some work to massage the data into a consolidated format before I start rendering.

**Pull request merged**